test/core/security/BUILD

```diff
@@ -29,6 +29,8 @@
 
 licenses(["notice"])  # 3-clause BSD
 
+package(default_visibility = ["//test/cpp:__subpackages__"],)
```

Comment:
instead of this could you just add a visibility = ["//test:__subpackages__"], to the target you used? That is oauth2_utils.

I do not feel strongly about this but this might make it easier to re-interpret it in our override of the cc_test rule.